### DISCUSSION TITLE
Autocomplete Debug.Log "misses"

### DISCUSSION THREAD

**Discussion by nicmar** [community/community#38638] [Copilot Conversations]
I'm using copilot in Rider with Unity. Sometimes when it autocompletes a Debug.Log, it shows a suggestion, but when i press tab, it doesn't apply it. I have to press CMD-Z (undo) and then tab again, and it works the second time.

In the gif below, i press TAB when I see the suggestion, but it looks like i'm pressing ESC to cancel the suggestion.
After CMD-Z and TAB again, it works.

![copilot](https://user-images.githubusercontent.com/3955579/200738330-6f1937f7-eea4-4ebb-89f2-398df7b7a174.gif)

PS: Not sure if this is the right forum, but this is where I got from using the "Forum" link on https://plugins.jetbrains.com/plugin/17718-github-copilot

**Comment by ossanna16** [comment]
Hi there @nicmar and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by iamcjwilliams** [comment]
@nicmar Thank you, I will make sure our copilot team investigates this!
